scripts/travisci_run.sh

```diff
@@ -15,7 +15,7 @@ export TERM=dumb
 # Go tests in Travis
 ./bin/buck test \
   --num-threads=3 \
-  --test-selectors '!GoBinaryIntegrationTest'
+  --test-selectors '!GoBinaryIntegrationTest,!GoTestIntegrationTest'
```

Comment:
This did not work as expected (the build failed with both of these failing).  Based on the help text, I think you can just pass these both like so: `--test-selectors '!GoBinaryIntegrationTest' '!GoTestIntegrationTest'`